ansible_collections/arista/avd/examples/cv-pathfinder/group_vars/WAN/management.yml

```diff
@@ -0,0 +1,59 @@
+---
+# WAN Management settings
+mgmt_interface_vrf: MGMT
+default_mgmt_method: oob
+mgmt_gateway: 192.168.17.1
+
+# NTP
+ntp_settings:
+  server_vrf: MGMT
+  servers:
+    - name: 0.pool.ntp.org
+
+# CloudVision/TerminAttr
+cvp_instance_ips: [apiserver.cv-staging.corp.arista.io]
+cvp_token_file: /mnt/flash/cv-onboarding-token
```

Comment:
token is the default for cvaas and apiserver will lead to issues.
```suggestion
cvp_instance_ips: [www.cv-staging.corp.arista.io]
```

Reply:
thats not what CVaaS is saying to use as IP

Reply:
![Screenshot 2024-09-10 at 16 42 17](https://github.com/user-attachments/assets/74541b99-8f57-43e9-b26d-4e516443e56c)

Reply:
ah right. Thanks